actions/ma-epd-ei-fiat.vbs

```diff
@@ -834,83 +835,115 @@ EMReadScreen elig_type_check_fourth_month, 2, 12, 50
 EMReadScreen elig_type_check_fifth_month, 2, 12, 61
 EMReadScreen elig_type_check_sixth_month, 2, 12, 72
 
-'Script will end if not DP
-If elig_type_check_first_month <> "DP" or elig_type_check_second_month <> "DP" or elig_type_check_third_month <> "DP" or elig_type_check_fourth_month <> "DP" or elig_type_check_fifth_month <> "DP" or elig_type_check_sixth_month <> "DP" then script_end_procedure("Not all of the months of this case are MA-EPD. Process manually.")
+'There needs to be at least 1 month of MA-EPD Elig Results - the script will check each month and if all are NOT DP - the script will end.
+If elig_type_check_first_month <> "DP" and elig_type_check_second_month <> "DP" and elig_type_check_third_month <> "DP" and elig_type_check_fourth_month <> "DP" and elig_type_check_fifth_month <> "DP" and elig_type_check_sixth_month <> "DP" then script_end_procedure("Not all of the months of this case are MA-EPD. Process manually.")
```

Comment:
Script end procedure should be updated to say no MA-EPD exists since the update is to account for partial DP budgets.

Reply:
Good call - updated.